lib/resolvers.js

```diff
@@ -29,3 +30,18 @@ export const mapLinkList = fieldName => doc => {
     return linkItem;
   }).filter(l => l !== undefined);
 };
+
+export const mapScheduleList = fieldName => doc => {
+  const schedule = doc[fieldName];
+
+  if (!schedule) { return []; }
+  return schedule.map((item) => {
+    if (!item.hasOwnProperty('time') || !item.hasOwnProperty('text')) {
+      return undefined;
+    }
+    const scheduleItem = {};
+    scheduleItem.time = pathOr('', ['time', 'value'], item);
+    scheduleItem.text = pathOr('', ['text', 'value'], item);
```

Comment:
Are we really considering it valid without text or time? Shouldn't both be needed?